graspologic/layouts/auto.py

```diff
@@ -249,6 +245,27 @@ def _node2vec_for_layout(
     return graph, tensors, labels
 
 
+def _to_undirected(graph: nx.DiGraph) -> nx.Graph:
```

Comment:
weights are sometimes not in the "weight" attribute - you shouldn't rely on this, in case the weight is actually in a field called "count" or something like that.

Reply:
I'll add a parameter 'weight_attribute' like the one in `is_real_weighted`. In that case, I would have to add that as an additional parameter for both `layout_umap` and `layout_tsne` too, right?

Reply:
Correct!